src/continuous_agg.c

```diff
@@ -783,3 +783,51 @@ ts_continuous_agg_get_user_view_oid(ContinuousAgg *agg)
 		elog(ERROR, "could not find user view for continuous agg");
 	return view_relid;
 }
+static int32
+find_raw_hypertable_for_materialization(int32 mat_hypertable_id)
+{
+	short count = 0;
```

Comment:
why a `short` and not `int`?

Reply:
doesn't really matter here. This should be <= 1 after the search.